.eslintrc.js

```diff
@@ -88,6 +88,7 @@ module.exports = {
       extends: ['plugin:@typescript-eslint/recommended', 'prettier'],
       rules: {
         'no-shadow': 'off',
+        'import/no-nodejs-modules': 'off',
```

Comment:
Guessing you're wanting to turn this off for server-side scripts? If so, I'd recommend creating a more targetted override set. You've added this one into `['**/*.{ts,tsx}']`, which includes files that can be run client-side, where node modules shouldn't be run.

Reply:
Okay, sounds good. How do I do this?

Should I just do this for `['config/**/*.ts','script/**/*.ts']`?

Reply:
yeah that sounds good for now, and can be adjusted later if needed 👍